phpBB/adm/style/acp_profile.html

```diff
@@ -17,7 +17,7 @@
 		</div>
 	<!-- ENDIF -->
 
-	<form id="add_profile_field" method="post" action="{U_ACTION}">
+	<form id="add_profile_field" method="post" action="{U_ACTION}"{S_FORM_ENCTYPE}>
```

Comment:
Then better yet, you can use `<!-- IF S_FORM_ENCTYPE is defined and S_FORM_ENCTYPE is not empty --> enctype="multipart/form-data"<!-- ENDIF -->` as `text/plain` would not be used ever anyway, and the other one is the defualt one.